dotcom-rendering/src/lib/verticalDivider.ts

```diff
@@ -22,9 +23,10 @@ export const verticalDivider = (
 				width: 1px;
 				height: 100%;
 				border-left: 1px solid
-					${containerOverrides
-						? containerOverrides.border?.container
-						: border.secondary};
+					${verticalDividerColour ??
+					(containerOverrides
+						? containerOverrides.border.container
+						: border.secondary)};
```

Comment:
So you can override the override?

Reply:
Probably worth pulling up one level and only passing the borderColour as to the `verticalDivider` method.

```ts
// capture the logic in one fell swoop and simplify the signatures of leaf methods
const borderColour =
	verticalDividerColour ??
	(containerPalette &&
		decideContainerOverrides(containerPalette).border.container &&
		palette.neutral[86]);
```